src/HyperFragmentView.tsx

```diff
@@ -25,14 +25,31 @@ var HyperFragmentViewManager: any;
 if (Platform.OS === 'android') {
   HyperFragmentViewManager = requireNativeComponent('HyperFragmentViewManager');
 }
+else {
+  HyperFragmentViewManager = requireNativeComponent('HyperFragmentViewManagerIOS');
+}
 
 const createFragment = (viewId: number, namespace: string, payload: string) => {
-  UIManager.dispatchViewManagerCommand(
-    viewId,
-    //@ts-ignore
-    UIManager.HyperFragmentViewManager.Commands.process.toString(),
-    [viewId, namespace, payload]
-  );
+  if (Platform.OS == 'android') {
+    UIManager.dispatchViewManagerCommand(
+      viewId,
+      //@ts-ignore
+      UIManager.HyperFragmentViewManager.Commands.process.toString(),
+      [viewId, namespace, payload]
+    );
+  }
+  else {
+    const commandId = UIManager.getViewManagerConfig(
+      'HyperFragmentViewManagerIOS'
```

Comment:
@TanishqJus - Let's change this to HyperFragmentViewManager to match android. You might need to change class name as well on the native side.